Library/ViewModels/NoShippingPostCampaignCheckoutViewModel.swift

```diff
@@ -34,12 +34,11 @@ public protocol NoShippingPostCampaignCheckoutViewModelOutputs {
   var goToLoginSignup: Signal<(LoginIntent, Project, Reward), Never> { get }
   var paymentMethodsViewHidden: Signal<Bool, Never> { get }
   var processingViewIsHidden: Signal<Bool, Never> { get }
-  var showErrorBannerWithMessage: Signal<String, Never> { get }
+  var showErrorBannerWithMessage: Signal<(String, /* persist: */ Bool), Never> { get }
```

Comment:
nit: Adding parameter names instead of an inline comment might clarify that this output has the option to persist.